frame/nomination-pools/src/lib.rs

```diff
@@ -1314,6 +1329,11 @@ pub mod pallet {
 	pub type ReversePoolIdLookup<T: Config> =
 		CountedStorageMap<_, Twox64Concat, T::AccountId, PoolId, OptionQuery>;
 
+	/// Map from a pool member account to their preference regarding reward payout
+	#[pallet::storage]
+	pub type RewardClaimPermission<T: Config> =
+		StorageMap<_, Twox64Concat, T::AccountId, RewardClaim, ValueQuery>;
```

Comment:
Shouldn't we use a cryptographic hasher (Blake2) here instead of Twox?
Context: https://substrate.stackexchange.com/questions/4245/storagehasher-blake2-vs-twox